elfo-logger/src/actor.rs

```diff
@@ -133,48 +134,54 @@ impl Logger {
 
         // <timestamp> <level> [<trace_id>] <object> - <message>\t<fields>
 
```

Comment:
In the vast majority of cases, the limit will not be reached, so I expect no performance penalty at all in such cases.

I think it better to wrap this code and call it twice:
1. With `payload=fields=buffer`
2. Check limit
3. Rerun with dedicated `payload`, `fields`, and `buffer` parts if needed

Reply:
I suppose it's done by bd8d9cffbdc6453afa3642844ba160dd7c2755fe